src/rfbzero/experiment.py

```diff
@@ -120,41 +335,91 @@ def run(
             degradation: DegradationMechanism = None,
             cls_degradation: DegradationMechanism = None,
             ncls_degradation: DegradationMechanism = None,
-            cross_over: Crossover = None,
+            crossover: Crossover = None,
     ) -> CyclingProtocolResults:
-        """Applies a cycling protocol and (optional) degradation mechanisms to a cell model"""
+        """
+        Applies a cycling protocol and (optional) degradation mechanisms to a cell model.
+
+        Parameters
+        ----------
+        duration : int
+            Simulation time (s).
+        cell_model : ZeroDModel
+            Defined cell parameters for simulating.
+        degradation : DegradationMechanism, optional
+            Degradation mechanism(s) applied to CLS and NCLS.
+        cls_degradation : DegradationMechanism, optional
+            Degradation mechanism(s) applied to CLS.
+        ncls_degradation : DegradationMechanism, optional
+            Degradation mechanism(s) applied to NCLS.
+        crossover : Crossover, optional
+            Crossover mechanism applied to cell.
+
+        """
         raise NotImplementedError
 
-    def _init_protocol(
+    @staticmethod
+    def _validate_cycle_values(
+            value: float,
+            value_charge: float,
+            value_discharge: float,
+            name: str
+    ) -> tuple[float, float]:
+        if value is not None and (value_charge is not None or value_discharge is not None):
```

Comment:
need additional check to throw error where value is None but user only input charge (or discharge)